packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -3120,21 +3159,34 @@ export class ContainerRuntime
 		address: string | undefined,
 		type: string,
 		content: any,
+		targetClientId?: string,
 	): ISignalEnvelope {
-		const newSequenceNumber = ++this._perfSignalData.signalSequenceNumber;
+		const isTargetedSignal = targetClientId !== undefined;
+		const newSignalSequenceNumber = ++this._perfSignalData.signalSequenceNumber;
```

Comment:
Let's move this out of perfSignalData as we are now using it all of the time.

Reply:
I think its usage has stayed the same, if we're referring to signalSequenceNumber.